src/client/wallet/UserWalletTransactions.less

```diff
@@ -2,7 +2,7 @@
 @import (reference) "../styles/modules/variables.less";
 
 .UserWalletTransactions {
-  background-color: @white;
+  /*background-color: @white;*/
```

Comment:
why is this commented out?

Reply:
The newly added text also got the background which looks odd, hence moved the background to 
.UserWalletTransactions__transaction

Reply:
With these change, the UI would something like this: 

![image](https://user-images.githubusercontent.com/14275525/34001793-f400f412-e116-11e7-9269-7814c2fbacb6.png)

Just in time, as you can see in screenshot, I got a suspicious link through memo.

Reply:
ahh ok, but instead of commenting out the styles, you should just remove it. makes no sense to comment out the styles if it will not be used.

Also @bonustrack has the final say on the design, so he should review this :)

Reply:
Yes, that was my initial thought to remove it instead of commenting out. Anyway I'll remove it, it makes the code look much cleaner.